app/assets/javascripts/utilities/buildCommentHTML.js.erb

```diff
@@ -135,8 +135,12 @@ function react(comment) {
   var reactButton = `<button class="crayons-btn crayons-btn--ghost crayons-btn--icon-left crayons-btn--s mr-1 reaction-like inline-flex reaction-button" id="button-for-comment-${ comment.id }" data-comment-id="${ comment.id }">
     ${ iconSmallHeart }
     ${ iconSmallHeartFilled }
-    <span class="reactions-count" id="reactions-count-${ comment.id }">${ num }</span>
-    <span class="reactions-label hidden m:inline-block">like</span>
+    ${
+      i18next.t('comments.num_likes', {
```

Comment:
This is not an ideal composition because it does not allow a language where the `likes` part wraps around `num`, but since the element gets update programmatically IIRC, we'd need to rewire the logic a bit.